vg/vgimg/vgimg.go

```diff
@@ -161,7 +168,20 @@ func UseImageWithContext(img draw.Image, gc draw2d.GraphicContext) option {
 		c.img = img
 		c.gc = gc
 		c.dpi = gc.GetDPI()
-		return setsDPI | setsSize
+		return setsDPI | setsSize | setsBackground
+	}
+}
+
+// UseBackgroundColor specifies the image background color.
+// The default color is white.
+func UseBackgroundColor(c color.Color) option {
+	return func(canvas *Canvas) uint32 {
+		// Add this operation as a initHook so that it happens after
+		// canvas.img is initialized.
+		canvas.initHooks = append(canvas.initHooks, func(canvas *Canvas) {
```

Comment:
This is subtle. It think it needs clearer commentary explaining that this operation happens on the first loop over the opts to queued for the second.